docs/configuration/configuration.md

```diff
@@ -474,6 +471,42 @@ tags:
 # The time after which the provided names are refreshed.
 # On large setup it might be a good idea to increase this value because the catalog will change all the time.
 [ refresh_interval: <duration> | default = 30s ]
+
+# Authentication information used to authenticate to the API server.
```

Comment:
```suggestion
# Authentication information used to authenticate to the consul server.
```

Reply:
Whoops I stole this directly from HTTP SD